### ISSUE TITLE
关于OLI2MSI 数据集的提问

### ISSUE THREAD

**Issue by yaofulin98** [zxk688/GCRDN#1]
I am very interested in your project. You mentioned in your paper using Landsat 8 OLI images with a spatial resolution of 30 meters as LR and Sentinel-2 MSI images with a spatial resolution of 10 meters as HR. Does that mean these sets of images are one-to-one correspondences? If possible, could you please share your dataset? Thank you very much.

**Comment by yaofulin98** [comment]
I found a similar dataset called OLI2MSI on Github, but the resolution of the images is somewhat different from what you mentioned in the article. Is your dataset also cropped based on this？

**Comment by littlebeen** [comment]
Yes, we crop the images for training. We use the OLI2MSI you find on the Github. However, the clipping does not influence the spatial resolution of the image. It just influences the size of the images instead of the spatial resolution.
